app/views/admin/display_ads/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+  <%= label_tag :organization_id, "Organization ID:" %>
+  <%= text_field_tag :organization_id, @display_ad.organization_id, class: "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :body_markdown, "Body Markdown:" %>
+  <%= text_area_tag :body_markdown, @display_ad.body_markdown, size: "100x10", class: "form-control" %>
+</div>
+
+<div class="form-group">
+  <%= label_tag :placement_area, "Placement area:" %>
```

Comment:
_Quick, non-blocking suggestion_: I think this would look nice titlecased since all of the other labels are. 😄 
```suggestion
  <%= label_tag :placement_area, "Placement Area:" %>
```